engine/model.py

```diff
@@ -341,6 +353,39 @@ def _prevent_oob(self) -> None:
 
         self.set_player_speed(new_vx, new_vy)
 
+    def handle_collisions_with_solids(self) -> None:
```

Comment:
Make this method private by putting an underscore at the start of the name: `_handle_collisions_with_solids`.